taskvine/src/manager/vine_current_transfers.c

```diff
@@ -214,16 +214,26 @@ int vine_current_transfers_wipe_worker(struct vine_manager *q, struct vine_worke
 		return removed;
 	}
 
+	char **ids_to_remove = malloc(hash_table_size(q->current_transfer_table) * sizeof(char *));
```

Comment:
I'd suggest using a `struct list` instead. It is what we do in the rest of code and will ease the life of future developers.